### PR TITLE
chore(all): update template, update tests

### PR DESCRIPTION
**Pull request by m1ga**
* remove all mobileweb, tizen, windows, blackberry parts
* move "ti.appcelerator.org" to "ti.tide.io" (it's overwriting the tisdk templates again!)
* fix tests

### CODE REVIEW

**Comment by m1ga** [comment]
Stupid Emulator tests :-) But now it's working again

**Comment by cb1kenobi** [comment]
This is a major breaking change. Can you bump the version in the `package.json`?

File: package.json
Comment (cb1kenobi) [inline package.json:9]: Let's do a major release.

```suggestion
  "version": "6.0.0",
```

While you're updating the `package.json`, could you please set the `"author": "TiDev, Inc. <npm@tidev.io>"` and remove the `maintainers`.

We really should bump the min Node.js version to Node 16. Even then, we should drop Node 16 and 18. Ugh, maybe a larger effort. Dunno.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: package.json
Comment (m1ga) [reply package.json]: sure, done :+1: I think if we do a major update we should merge https://github.com/tidev/node-titanium-sdk/pull/652 too so the Arm Mac users are happy again :smile:

**Review by cb1kenobi — APPROVED** [review]

File: package.json
Comment (cb1kenobi) [reply package.json]: Ok, I'll merge this and wait to publish until #652 has been reviewed/merged.

**Pull request merged**
